sys/include/net/dtls.h

```diff
@@ -44,7 +44,7 @@ extern "C" {
  * @brief The maximum number DTLS peers (i.e. sessions)
  */
 #ifndef CONFIG_DTLS_PEER_MAX
-#define CONFIG_DTLS_PEER_MAX       (1)
+#define CONFIG_DTLS_PEER_MAX       (2)
```

Comment:
After discussion on Matrix, I've updated the default to only change if the GCOAP_TLS module is in use (in which case there's always a server running).

If gcoap ever becomes usable client-only, this can be made more precise; also, if Gcoap's CONFIG_GCOAP_DTLS_MINIMUM_AVAILABLE_SESSIONS mechanism becomes smarter and only kicks when someone else tries to get in, this could be set lower again (at the cost of allowing any peer the DoS attack of terminating DTLS connections).